src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -251,14 +251,10 @@ export function useHostnameValidation(): StepValidation {
 
 export function useKernelValidation(): StepValidation {
   const kernel = useAppSelector(selectKernel);
+  const errors = {};
 
   if (!isKernelNameValid(kernel.name)) {
-    return {
-      errors: {
-        kernel: 'Invalid format.',
-      },
-      disabledNext: true,
-    };
+    Object.assign(errors, { kernel: 'Invalid format.' });
```

Comment:
I'm not opposed to this but it's a somewhat new pattern... what's the advantage of using `Object.assign()`?

Reply:
I've started using `Object.assign()` on steps that have multiple validated fields. The main reason is that we need to return an object with more than one possible error values.

In this case we were previously returning either
```typescript
errors: { kernel: ... }
```

OR 
``` typescript
errors: { kernelAppend: ... }
```

but there was no pattern for cases when both imported values are invalid at the same time AFAIK. This way we can just keep adding possible problems into the `errors` object and then return the entire thing at once.